lib/grantinee/cli.rb

```diff
@@ -108,9 +108,19 @@ def process_require_param
 
     # Database configuration file
     def process_database_param
-      require options[:config] if options[:config]
+      unless @options[:config] || Grantinee.configuration.configured?
+        Grantinee::Engine.detect_active_record_connection!
 
-      Grantinee::Engine.detect_active_record_connection! unless Grantinee.configuration.configured?
+        unless Grantinee.configuration.configured?
+          raise "No configuration file found. Please use the -c option"\
```

Comment:
I am not a fan of these broken lines, no matter what Rubocop writes about them ;-)

Reply:
haha besides rubocop, I like limiting the width of the code to 100 characters max, I think it keeps things more readable :)